_includes/color-matrix.html

```diff
@@ -9,7 +9,7 @@
   <svg><use xlink:href="#usa-matrix-bad-contrast-ratio"/></svg>
 
   <p class="usa-sr-invisible" aria-hidden="true">
-    <strong>Don't</strong> use these color combinations. They don’t have enough contrast to meet accessibility standards, and some people will have difficulty reading the text.
+    <strong>Don’t</strong> use these color combinations. They don’t have enough contrast to meet accessibility standards, and some people will have difficulty reading the text.
```

Comment:
I should have started that with an "I'm not certain about this, but I've heard that..."